command/ca/provisionerbeta/update.go

```diff
@@ -794,3 +815,32 @@ func updateGCPDetails(ctx *cli.Context, p *linkedca.Provisioner) error {
 	}
 	return nil
 }
+
+func updateSCEPDetails(ctx *cli.Context, p *linkedca.Provisioner) error {
+	data, ok := p.Details.GetData().(*linkedca.ProvisionerDetails_SCEP)
+	if !ok {
+		return errors.New("error casting details to SCEP type")
+	}
+	details := data.SCEP
+
+	if ctx.IsSet("force-cn") {
+		details.ForceCn = ctx.Bool("force-cn")
+	}
+	if ctx.IsSet("challenge") {
+		details.Challenge = ctx.String("challenge")
+	}
+	if ctx.IsSet("capabilities") {
```

Comment:
Are you intending to replace the entire list with the new one? For a few of the other provisioners I had to define a `remove-<flag>` to have a list of the values to remove and then do the logic in the update method before post-ing. Might no be necessary for this, just letting you know.

Reply:
I saw your previous work and thought about adding those, but decided against it (for now). There'll likely be more work in the future related to SCEP and that might also touch the capabilities. Making this part a bit more user friendly can be done then, I think.